source/OutfitInfoDisplay.cpp

```diff
@@ -334,7 +344,17 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 		outfit.RelativeEnergyDamage() * 100.,
 		outfit.FiringEnergy(),
 		outfit.FiringHeat(),
-		outfit.FiringFuel()
+		outfit.FiringFuel(),
+		outfit.FiringHull(),
+		outfit.FiringShields(),
+		outfit.FiringIon(),
+		outfit.FiringSlowing(),
+		outfit.FiringDisruption(),
```

Comment:
Just glancing right now. Status effects should be multiplied by 100.
```suggestion
		outfit.FiringIon() * 100.,
		outfit.FiringSlowing() * 100.,
		outfit.FiringDisruption() * 100.,
```